typings/index.d.ts

```diff
@@ -1484,25 +1484,25 @@ declare module 'klasa' {
 		| GuildChannel
 		| Snowflake;
 
-	export type SchemaFolderAddOptions = SchemaFolderOptions | SchemaPieceOptions;
+	export type SchemaFolderAddOptions = SchemaFolderOptions | SchemaEntryOptions;
 
-	export type SchemaPieceOptions = {
+	export type SchemaEntryOptions = {
 		array?: boolean;
 		configurable?: boolean;
 		default?: any;
 		min?: number;
 		max?: number;
-		filter?: ((client: KlasaClient, value: any, schema: SchemaPiece, language: Language) => boolean) | null;
+		filter?: ((client: KlasaClient, value: any, schema: SchemaEntry, language: Language) => boolean) | null;
```

Comment:
schema should probably be changed here as well, it's not the full schema, just the entry (parameter name)